src/main/java/uk/gov/ons/census/fwmtadapter/messaging/ActionFieldReceiver.java

```diff
@@ -54,6 +54,10 @@ public void receiveMessage(FieldworkFollowup followup) {
     actionInstruction.setUprn(followup.getUprn());
     actionInstruction.setUndeliveredAsAddress(followup.getUndeliveredAsAddress()); // TODO: Delete?
 
+    if (followup.getAddressType().equals("CE")) {
```

Comment:
Not keen on building this stuff in a way specific to CE's but I won't block it as a first iteration.

Reply:
Just reading the same bit, feels to me like it should just set it for all

Reply:
read the FWMT specs - this field doesn't exist on HH messages. We need to either conform to specs or get them changed.

Reply:
cool, if it's in the specs, it's in the specs.  I think they're OTT, but that's another story.